generic/tests/cfg/timedrift.cfg

```diff
@@ -19,11 +19,16 @@
                     used_cpus = 100
                     variants:
                         - default_load:
+                            host_load_instances = 0
+                            guest_load_instances = 2
```

Comment:
Is there some standard to explain why guest has **2** load instances in the `default_load` scenario?

Reply:
Thanks, not yet, no standard for default load.

Reply:
Hi @luckyh , there is no standard in polarion case for `default_load`, I choose 2 instances because I think half of guest cpus (at least 4 cpus) can run load as default load when we test guest's time drift,  what do you think?